src/subcommand.rs

```diff
@@ -85,8 +85,24 @@ impl Subcommand {
 
     /// Reads gz mrt data from urls defined by range, decompresses them, parses mrt output, finds bottleneck.
     fn find_bottleneck(dump: &PathBuf, out: Option<&Path>) -> Result<()> {
-        let bottleneck = FindBottleneck::locate(dump)?;
-        bottleneck.write(out)?;
+        // Until the work is done, store results in this file so that
+        // a disrupted process doesn't look finalized.
+        let temp_result_file = NamedTempFile::new();
```

Comment:
This type has some weird properties, see [docs](https://docs.rs/tempfile/3.1.0/tempfile/struct.NamedTempFile.html).
But regular tempfile doesn't allow reading from it it seems. This is nonsense, why would someone create a file which can't be read, but only written in?

Reply:
We need to fix this.